prow/config.yaml

```diff
@@ -40,6 +40,7 @@ deck:
   tide_update_period: 1s
   hidden_repos:
   - kubernetes-security
+  google_analytics: UA-82843984-5
```

Comment:
is this the same analytics account used for the rest of our stuff? cc: @castrojo

Reply:
It's the same as the rest of the sig-testing stuff.